canvas_manage_course/settings/base.py

```diff
@@ -103,8 +93,7 @@
     'async': 'default',
     'auth': 'default',
     'contenttypes': 'default',
-    'icommons_common': 'termtool',
-    'lti_permissions': 'default',  # deprecated, but still around for migrations
+    'icommons_common': 'coursemanager',
```

Comment:
Sure - as an aside, I think we should update that CourseSchema router so that it defaults to the coursemanager schema instead of termtool at some point.